mrp_unbuild_cust_alu_analytics_maintenance/models/maintenance_equipment.py

```diff
@@ -0,0 +1,19 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import _, api, fields, models
+
+class MaintenanceEquipment(models.Model):
+    _inherit = "maintenance.equipment"
+
+    works_with_unbuild = fields.Boolean(string='Works with Unbuild', default=False)
+
+    unbuild_count = fields.Integer(
+        compute="_compute_count_unbuilds", string="Unbuild Incidences"
+    )
+
+    def _compute_count_unbuilds(self):
+        for record in self:
+            incidence_ids = self.env["mrp.unbuild.incidence"].search([("maintenance_equipment_id", "=", record.id)])
+            unbuild_ids = incidence_ids.mapped("unbuild_id")
+            record.unbuild_count = len(unbuild_ids)
```

Comment:
This code is right, but still inefficient. Since this field is only computed in a form, is fast (for a single record). But, if we should add it to e.g. a tree view it should become very slow.
Please simply add a TODO comment for a later improvement of this (we could replace it by a `read_group` strategy)

Reply:
Should I use read_group allways for this cases, since with search function, it never will be efficient enought with multiple records?

Reply:
For non-stored count compute fields, the rule should be as follows:
- if count is only accessed within a form view, "len()" strategy should be enough. Or `search_count()` as well. In both cases, compute method is called with a single record, so no matter of the strategy, it should be fast (but be careful, a form access could be only in early development status and later added other access that make better the below strategy).
- if count is accessed from other views (tree, kanban, and others), `read_group()` should be the best option. There are a lot of examples in OCA/solvosci.